jupyter-book/introduction/interoperability.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<p>Maybe also list common data types that rpy2 works with? like panda's dataframe, etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/single-cell-best-practices/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>